### ISSUE TITLE
Which models to include?

### ISSUE THREAD

**Issue by TimTaylor** [reconverse/trending#2]
Currently `lm`, `glm`, `MASS::glm.nb`, `brms::brm`

**Comment by dirkschumacher** [comment]
In addition I would suggest glmnet. Anything that can do regularized regression

**Comment by tjtnew** [comment]
Sounds good. Not used glmnet but it looks like we will probably need glmnetutils for a the formula interface.

**Comment by dirkschumacher** [comment]
Oh yeah, that is probably necessary, as they have a pure matrix based interface

**Comment by tjtnew** [comment]
Also, not that it's a problem, I don't think we could add confidence/prediction intervals for glmnet.

**Comment by dirkschumacher** [comment]
Yeah, of course we need to research proper prediction intervals for regularized regression.
